pytorch_blade/src/ltc/init_python_bindings.cpp

```diff
@@ -16,10 +16,8 @@
 namespace torch_disc {
 namespace py = pybind11;
 void InitLtcModuleBindings(py::module m) {
-  m.def("_ltc_init_disc_backend", []() { compiler::InitTorchScriptBackend(); });
-}
-
-PYBIND11_MODULE(_torch_disc, m) {
-  torch_disc::InitLtcModuleBindings(m);
+  py::module ltc =
+      m.def_submodule("_ltc", "torch_blade python bindings to mlir");
```

Comment:
Nit: python bindings to lazy tensor core?

Reply:
done